app/org/sagebionetworks/bridge/services/UploadService.java

```diff
@@ -236,12 +259,10 @@ public Upload getUpload(@Nonnull String uploadId) {
             throw new BadRequestException("Query window cannot be longer than two days: " + startTime + "-" + endTime);
         }
```

Comment:
It would be preferable to do something like "startTime.plusDays(2).isBefore(endTime)". Otherwise, really weird things happen with leap years and leap seconds.